intercom_flutter_platform_interface/test/method_channel_intercom_flutter_test.dart

```diff
@@ -330,6 +330,7 @@ void main() {
 
       setUp(() {
         channel.setMockMethodCallHandler((MethodCall methodCall) async {
+          // ignore: unnecessary_non_null_assertion
           ServicesBinding.instance!.defaultBinaryMessenger
```

Comment:
With Flutter 3.0.0 stable version this (!) is not required with `ServicesBinding.instance`. For now better to ignore this warning as not all users have updated to 3.0.0

Reply:
Can you add a TODO to fix the ignore?